base/initdefs.jl

```diff
@@ -25,8 +25,8 @@ Stop the program with an exit code. The default exit code is zero, indicating th
 program completed successfully. In an interactive session, `exit()` can be called with
 the keyboard shortcut `^D`.
 """
-exit(n) = ccall(:jl_exit, Cvoid, (Int32,), n)
-exit() = exit(0)
+exit(n; wait_task::Bool=false) = ccall(:jl_exit, Cvoid, (Int32, Int32), n, wait_task)
```

Comment:
naming bikeshedding time: should we say this is `exit(0, process=true)`, since it specifies whether this call just kills this task or all tasks?

Reply:
I went with `kill_tasks=true`.